### PR TITLE
1457-fewiki-replace-favMutation-in-favoritesIcon-component-with-corresponding-openapi-generated-fetcher

### LINKED ISSUE #1457: [FE][WIKI] Replace `favMutation` in FavoritesIcon component with corresponding openapi generated fetcher

### PR DESCRIPTION
**Pull request by Fanur1991**
closes #1457

### CODE REVIEW

File: web/wiki/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.23.2] - 2024-07-29
```
Comment (kevinmamaqi) [inline web/wiki/CHANGELOG.md:5]:
```suggestion
## [0.23.0] - 2024-07-29
```

File: web/wiki/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "web_wiki",
   "private": true,
-  "version": "0.22.2",
+  "version": "0.23.2",
```
Comment (kevinmamaqi) [inline web/wiki/package.json:4]:
```suggestion
  "version": "0.23.0",
```

**Review by kevinmamaqi — CHANGES_REQUESTED** [review]

**Review by Cristian-Moller — DISMISSED** [review]

**Review by zookatt — APPROVED** [review]

**Review by Cristian-Moller — APPROVED** [review]

File: web/wiki/package.json (hunk 2 above)
Comment (Fanur1991) [reply web/wiki/package.json]: Changed

File: web/wiki/CHANGELOG.md (hunk 1 above)
Comment (Fanur1991) [reply web/wiki/CHANGELOG.md]: Changed

**Pull request merged**